src/cloudevent/partials/alerts/app-distribution-on-new-tester-ios-device-published.ts

```diff
@@ -21,7 +21,7 @@ export const alertsAppDistributionOnNewTesterIosDevicePublished: MockCloudEventP
         createTime: now,
         endTime: now,
         payload: {
-          '@type':
+          ['@type']:
```

Comment:
FYI: the brackets here read a little funny. You only need the brackets if you need to evaluate the value in the brackets to get a real key name. E.g.

```
const keyName = '@type';

payload = {
  [keyName]: 'type.googleapis.com/...'
}
```